### ISSUE TITLE
update INSTALL.md with extra info for installation

### ISSUE THREAD

**Issue by cookie33** [pidconsortium/EPIC-API-v2#6]
INSTALL.md misses some info:
1. add which parameters to update in config.rb. And how:
     REALM
     OPAQUE
     SEQUEL_CONNECTION_ARGS
     LOG_SETTINGS
     NO_DELETE
     ENFORCED_PROFILES
     DEFAULT_GENERATOR
1. add which parameters to update in config.ru
   connection port for server
   rack reloader
   proxy server config
2. add which ports are needed for a firewall:
   incoming:
   outgoing:
3. We don't need ant to be installed.
4. for centos 7 we need other images than;

```
apache-tomcat-apis mod_ssl
```

We need;

```
tomcat-jsp-2.2-api tomcat-servlet-3.0-api mod_ssl
```
1. mysql-connector-java needs to be installed before it can be used.
2. an example of config.dct would be nice to have an example to check against.
3. to be safe/sure an certificate has to installed in the apache server
4. the document states that it start web brick server while default mizuno is started.

**Comment by cookie33** [comment]
if config.dct does not have:

```
    "case_sensitive" = "yes"
```

My tests to check if the installation went fine fail. So this is also something to add to the configuration.

**Comment by cookie33** [comment]
fixed by merge #7
